setup.sh

```diff
@@ -520,7 +520,7 @@ elif [ $task == "--install-neovim" ]; then
         xattr -c ./nvim-macos.tar.gz
         tar -xzf nvim-macos.tar.gz
         mkdir -p "$HOME/opt/bin"
-        mv nvim-macos-arm64 "$HOME/opt/neovim"
+        mv nvim-macos-* "$HOME/opt/neovim"
```

Comment:
Just to make it a little more specific:

```suggestion
        mv nvim-macos-*64 "$HOME/opt/neovim"
```